src/Plugins/BotSharp.Plugin.RoutingSpeeder/Controllers/RoutingSpeederController.cs

```diff
@@ -24,9 +25,17 @@ public RoutingSpeederController(IServiceProvider service)
     public IActionResult TrainIntentClassifier(TrainingParams trainingParams)
     {
         var intentClassifier = _service.GetRequiredService<IntentClassifier>();
-        intentClassifier.InitClassifer(trainingParams.Inference);
+        // intentClassifier.InitClassifer(trainingParams.Inference);
```

Comment:
Should we comment this for training?

Reply:
The InitClassifer will call LoadWeights method. If we still want to re-train model when some weights exist, calling this method will load that weights first and then continue training.